src/ui/pages/GameUI.jsx

```diff
@@ -14,16 +15,26 @@ const wrapper = css`
 `;
 
 const gameBorder = css`
+  position: relative;
   width: 100%;
   max-width: max(min(50vw, 1080px), 1076px);
   max-height: 1000vh;
-  overflow-y: auto;
   border: 1rem solid ${theme.colors.primary.main};
   display: flex;
   place-items: center;
   background: ${theme.colors.background.transparent};
 `;
```

Comment:
```suggestion
const gameBorder = css`
  position: relative;
  border: 1rem solid ${theme.colors.primary.main};
  margin-bottom: 4rem; /* lepiej: obliczyć wartość na bazie BUTTON_HEIGHT_SIZE */
`;
```

Przenosząc szerokość na `canvas` możesz mocno skrócić CSS: `width`/`max-width`/`max-height` - niepotrzebne bo box dopasuje się do dziecka. `display`, `place-items`, `background` - niepotrzebne, bo cały box jest wypełniony.